zmicer_zadorau/05.Ansible.start/1/inv.yaml

```diff
@@ -0,0 +1,31 @@
+---
+all_hosts:
```

Comment:
In yaml could you please use double withespace for nested lines

Reply:
I fixed the file zmicer_zadorau/05.Ansible.start/1/inv.yaml